javascript-conditionals/main.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-unreachable */
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(number) {
+  return number < 5;
+}
+
+function isEven(number) {
+  return number % 2 === 0;
+}
+
+function startsWithJ(string) {
+  return string[0] === 'J';
+}
+
+function isOldEnoughToDrink(person) {
+  return person.age >= 21;
+}
+
+function isOldEnoughToDrive(person) {
+  return person.age >= 16;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH > -1 && pH < 7) {
+    return 'acid';
+  } else if (pH > 7 && pH <= 14) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
+}
+
+function introduceWarnerBro(name) {
+  switch (name) {
+    case 'yakko':
+    case 'wakko':
+      return "We're the warner brothers!";
+      break;
+    case 'dot':
+      return "I'm cute~";
+      break;
+    default:
+      return 'Goodnight everybody!';
+  }
+}
+
+function recommendMovie(genre) {
+  switch (genre) {
+    case 'action':
+    case 'comedy':
+    case 'drama':
+    case 'musical':
+    case 'sci-fi':
+      return 'The Spongebob Squarepants Movie';
```

Comment:
While I agree that the Spongebob Squarepants Movie is a great movie and genre shifting haha, you should have individual movies for each of these cases

Reply:
Spongebob Squarepants Movie is the only movie you'll ever need but ok lol